QgisModelBaker/gui/options.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, configuration, parent=None):
         self.ili2db_command_reload()
 
         settings = QSettings()
-        drop_mode = DropMode(settings.value('QgisModelBaker/drop_mode', defaultValue=3))
+        drop_mode = DropMode(settings.value('QgisModelBaker/drop_mode', 3, int))
```

Comment:
```suggestion
        drop_mode = DropMode(settings.value('QgisModelBaker/drop_mode', DropMode.ask, int))
```

?

Reply:
I have to store the settings as it's value (integer) because I cannot stor Qt Objects (enums)...

Reply:
We could use

`DropMode.ask.value` (3) or even nicer `DropMode.ask.name` (yes)

Reply:
Another note, could we use `ASK` (uppercase) -> see https://mail.python.org/pipermail/python-ideas/2016-September/042340.html

Reply:
Yes. We store the Value like this:
```
settings.setValue('QgisModelBaker/drop_mode', DropMode.ask.value)
```
So i think 3 is stored, no?
And so we have to read 3 (as int) and then convert to the enum with `DropMode(3)`

Or what am I missing?